packages/mui-material/test/typescript/OverridableComponent.spec.tsx

```diff
@@ -7,7 +7,7 @@ interface MyOverrideProps {
   myString?: string;
   myCallback?(n: number): void;
 }
-declare const MyOverrideComponent: React.ComponentType<MyOverrideProps>;
+declare const MyOverrideComponent: React.FunctionComponent<MyOverrideProps>;
```

Comment:
Needs to be double checked, for some reason I needed to change the type from `ComponentType` to `FunctionComponent`.